internal/middleware/auth.go

```diff
@@ -56,4 +59,13 @@ func AuthenticateRebac(next http.Handler, jimm jujuapi.JIMM) http.Handler {
 		ctx = rebac_handlers.ContextWithIdentity(ctx, user)
 		next.ServeHTTP(w, r.WithContext(ctx))
 	}), jimm)
+
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		relativePath, _ := strings.CutPrefix(r.URL.Path, baseURL)
+		if relativePath == "/v1/swagger.json" {
```

Comment:
perhaps define that one endpoint as a constant.. or better maybe
```
var unauthenticatedEndpoints []string = []string{
   "/v1/swagger.json"
}

for _, path := range unauthenticatedEndpoints {
  if relativePath == path {
    next.ServeHTTP(w, r)
  }
}
```

Reply:
How about a map?